commands/init.go

```diff
@@ -37,14 +39,32 @@ func transformInitArgs(args *Args) error {
 		return nil
 	}
 
-	name, err := utils.DirName()
-	if err != nil {
-		return err
+	var (
+		name   string
+		newDir bool
+		err    error
+	)
+
+	if args.IsParamsEmpty() {
+		name, err = utils.DirName()
+		if err != nil {
+			return err
+		}
+	} else {
+		name = args.LastParam()
+		newDir = true
 	}
 
 	project := github.NewProject("", name, "")
 	url := project.GitURL("", "", true)
-	args.After("git", "remote", "add", "origin", url)
+
+	cmds := []string{"git"}
+	if newDir {
+		cmds = append(cmds, "--git-dir", filepath.Join(name, ".git"))
```

Comment:
`--separate-git-dir` is a flag for `git-init`. But the line is to run `git --git-dir DIR remote add origin URL` for `git-remote`